_episodes/07-vector-data-in-python.md

```diff
@@ -150,9 +156,7 @@ This array contains, in order, the values for minx, miny, maxx and maxy, for the
 We can convert these coordinates to a bounding box or acquire the index of the dataframe to access the geometry. Either of these polygons can be used to clip rasters (more on that later). 
 
 ## Selecting spatial features
-Our `cropfield` dataset is rather large, containing data for the entirety of the European portion of the Netherlands. Before plotting it we will first select a specific section to be our area of interest.
-
-We can create a cropped version of our dataset as follows:
+Sometimes, the loaded data can still be too large. We can cut it is to a even smaller extent using the `.cx` function:
```

Comment:
Don't know whether this is creating more confusion, but maybe it is needed to anticipate the question "why we use square brackets here?"? 
```suggestion
Sometimes, the loaded data can still be too large. We can cut it is to a even smaller extent using the `.cx` indexer (note the use of square brackets instead of round brackets, which are used instead with functions and methods):
```